source/common/config/well_known_names.h

```diff
@@ -128,11 +133,20 @@ class TagNameValues {
   // Returns the list of descriptors.
   const std::vector<Descriptor>& descriptorVec() const { return descriptor_vec_; }
 
+  // Returns the list of tokenized descriptors.
+  const std::vector<TokenizedDescriptor>& tokenizedDescriptorVec() const {
+    return tokenized_descriptor_vec_;
+  }
+
 private:
   void addRe2(const std::string& name, const std::string& regex, const std::string& substr = "");
+  void addTokenized(const std::string& name, const std::string& regex);
```

Comment:
Should we have some documentation for the format used here? Also the param name regex seems wrong in this context